### ISSUE TITLE
File `zref-abspage.sty' not found.

### ISSUE THREAD

**Issue by ewilan-riviere** [Wandmalfarbe/pandoc-latex-template#241]
I try to use this template, I had some packages to install but with someone, I've a problem

```bash
! LaTeX Error: File `zref-abspage.sty' not found.
```

I try to install it with

```bash
sudo tlmgr zref-abspage
```

But tlmgr not found this package, I don't know how to download it.

- MacOS Big Sur 11.2.3
- LaTeX and Pandoc with Homebrew

**Comment by TheTaoOfPhil** [comment]
I went through days of not being able to get the most basic pandoc options for  LaTeX working on my mac. Whatever I had installed from brew did not set any of the environment variables correctly. The solution was to install [MacTex](http://www.tug.org/mactex/). I had tried to use MicTex but dispite it having a mac-specific edition, it did not work. MacTex apparently knows how to get the LaTex install right on a mac. 

I also had issues getting the download from tug.org to work. Downlowing it fron the [ConcertPass Mirror](https://mirrors.concertpass.com/tex-archive/systems/mac/mactex/) finally worked. 

Best of luck.
